src/test/spec/versioned_api.rs

```diff
@@ -1,100 +1,12 @@
-use std::path::PathBuf;
+use tokio::sync::RwLockWriteGuard;
 
-use tokio::sync::{RwLockReadGuard, RwLockWriteGuard};
-
-use crate::{
-    bson::doc,
-    options::{ServerApi, ServerApiVersion},
-    test::{
-        log_uncaptured,
-        run_single_test,
-        run_spec_test_with_path,
-        spec::unified_runner::TestFile,
-        EventClient,
-        CLIENT_OPTIONS,
-        LOCK,
-    },
-};
+use crate::test::{run_spec_test, LOCK};
 
 use super::run_unified_format_test;
 
 #[cfg_attr(feature = "tokio-runtime", tokio::test)]
 #[cfg_attr(feature = "async-std-runtime", async_std::test)]
 async fn run() {
     let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await;
-    run_spec_test_with_path(&["versioned-api"], run_non_transaction_handling_test).await;
-}
-
-#[cfg_attr(feature = "tokio-runtime", tokio::test)]
-#[cfg_attr(feature = "async-std-runtime", async_std::test)]
-async fn run_transaction_handling_spec_test() {
```

Comment:
IIRC this test was originally split out from the rest so we could skip it on certain topologies, but it looks like that conditional skip was removed. Now that we're running this unconditionally there's no need to separate it from the rest of the versioned API tests.